science/gildas/Portfile

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
+# $Id: Portfile 153461 2016-10-01 10:24:47Z smaret@macports.org $
```

Comment:
We prefer no longer having the `$Id$` lines, especially not in expanded form, now that we're using git which doesn't support them.